src/libraries/System.Private.CoreLib/src/System/Threading/Tasks/Task.cs

```diff
@@ -2448,6 +2448,7 @@ public TaskAwaiter GetAwaiter()
         /// true to attempt to marshal the continuation back to the original context captured; otherwise, false.
         /// </param>
         /// <returns>An object used to await this task.</returns>
+        [Intrinsic]
         public ConfiguredTaskAwaitable ConfigureAwait(bool continueOnCapturedContext)
```

Comment:
ConfigureAwait can participate in JIT optimizations (not in this PR), thus needs to be matched by name.